src/utils/strings/englishConstants.tsx

```diff
@@ -748,6 +748,8 @@ const englishConstants = {
             </p>
         </>
     ),
+    WEAK_DEVICE:
+        "The device you're using is not powerful enough for the encryption of your photos. Please try to log in on desktop, or download the ente mobile app.",
```

Comment:
Thanks for remembering this change!

I feel we should go with the message I'd suggested though. This message is shorter but it doesn't say the same thing. The problem is not the device (because if the problem was the device the it won't work on the app either). The problem is the mobile browser.

Reply:
"The mobile web browser you're using is not powerful enough for the encryption of your photos. Please try to log in to ente on your computer, or download the ente mobile app."